architecture/design/docdb-change-data-capture.md

```diff
@@ -56,7 +56,7 @@ We need to provide the configuration values in the json format, the important on
 }
 ```
 
-For a list of complete parameters that can be configured see [Debezium Connector for YugabyteDB](../../docs/content/preview/integrations/cdc/debezium-for-cdc.md).
+For a list of complete parameters that can be configured see [Debezium Connector for YugabyteDB](../../docs/content/preview/integrations/cdc/debezium.md).
```

Comment:
Should point to the docs site rather than the source file. (Also, verify the links after changing...)

```suggestion
For a list of complete parameters that can be configured see [Debezium Connector for YugabyteDB](https://docs.yugabyte.com/preview/integrations/cdc/debezium/).
```